test/test_stokes.py

```diff
@@ -44,6 +46,24 @@
 
 # {{{ test_exterior_stokes
 
+def rnorm2(actx, x, y, discr=None, p=None):
```

Comment:
- Name arguments to designate one as reference?
- These look like two functions mashed into one with an `if`?

Reply:
> These look like two functions mashed into one with an if?

They were wearing a trenchcoat! How could you tell? :facepalm: 

Cleaned up in 45efe7b.